.github/ISSUE_TEMPLATE/Bug.md

```diff
@@ -0,0 +1,47 @@
+---
+name: 🐞 Bug Report
+about: Something is broken? 🔨
+---
+
+### Bug Report
+
+<!-- Fill in the relevant information below to help triage your issue. -->
+
+|    Q        |   A
+|------------ | ------
+| BC Break    | yes/no
```

Comment:
What does "BC Break" mean?

Reply:
This term could be useful during libraries break changes; Obviously if we take a user point of view, it does not really make sense ... useful. 😕

Reply:
Okay 👍